### PR TITLE
SBAT: fix some typos in the SBAT docs

### PR DESCRIPTION
**Pull request by pcmoore**
Signed-off-by: Paul Moore <pmoore2@cisco.com>

### CODE REVIEW

File: SBAT.md [hunk 1]
```diff
@@ -453,7 +453,7 @@ And the UEFI CA issues an update to SBAT which has:
 sbat,1
 shim,1
 grub,3
-grub.fedora,1
+grub.fedora,2
 ```
```
Comment (martinezjavier) [inline SBAT.md:457]: Right, the doc also says:

```
At this point, each vendor updates their grub builds, and updates the component_generation in .sbat to 1
```
and that also should be `2`. Could you please also fix that while being there ?

Comment (martinezjavier) [inline SBAT.md:457]: Right, the doc also says:

```
At this point, each vendor updates their grub builds, and updates the component_generation in .sbat to 1
```

and should be `2`. Mind to also fix that while being there ?

Comment (pcmoore) [reply SBAT.md:457]: Sure thing, I'll fix that now and do a force-push to the PR.

**Comment by pcmoore** [comment]
The inconsistent line length in SBAT.md is also annoying me, give me a few minutes to fix that and I'll add a second patch to this PR.

**Pull request merged**